experiment/simulation/js/animation-parallel-register.js

```diff
@@ -0,0 +1,807 @@
+{
+    TweenLite.to(textSelect, 0, { autoAlpha: 0 });
+}
+function selectTextVisible()
+{
+    TweenLite.to(textSelect, 0, { autoAlpha: 1 });
+}
+function serialDotDisappear() {
+    TweenLite.to(serialDot1, 0, { autoAlpha: 0 });
+    TweenLite.to(serialDot2, 0, { autoAlpha: 0 });
+    TweenLite.to(serialDot3, 0, { autoAlpha: 0 });
+    TweenLite.to(serialDot4, 0, { autoAlpha: 0 });
+}
+
+function clockDotDisappear() {
+    TweenLite.to(clockDot, 0, { autoAlpha: 0 });
+
+}
+function serialDotVisible() {
+    TweenLite.to(serialDot1, 0, { autoAlpha: 1 });
+    TweenLite.to(serialDot2, 0, { autoAlpha: 1 });
+    TweenLite.to(serialDot3, 0, { autoAlpha: 1 });
+    TweenLite.to(serialDot4, 0, { autoAlpha: 1 });
+}
+
+function clockDotVisible() {
+    TweenLite.to(clockDot, 0, { autoAlpha: 1 });
+
+}
+function outputDisappear() {
+    TweenLite.to(textO1, 0, { autoAlpha: 0 });
+    TweenLite.to(textO2, 0, { autoAlpha: 0 });
+    TweenLite.to(textO3, 0, { autoAlpha: 0 });
+    TweenLite.to(textO4, 0, { autoAlpha: 0 });
+}
+function outputVisible() {
+    TweenLite.to(textO1, 0, { autoAlpha: 1 });
+    TweenLite.to(textO2, 0, { autoAlpha: 1 });
+    TweenLite.to(textO3, 0, { autoAlpha: 1 });
+    TweenLite.to(textO4, 0, { autoAlpha: 1 });
+}
+function serialDisappear1() {
+    TweenLite.to(textI1, 0, { autoAlpha: 0 });
+}
+function serialDisappear2() {
+    TweenLite.to(textI2, 0, { autoAlpha: 0 });
+}
+function serialDisappear3() {
+    TweenLite.to(textI3, 0, { autoAlpha: 0 });
+}
+function serialDisappear4() {
+    TweenLite.to(textI4, 0, { autoAlpha: 0 });
+}
+function clockDisappear() {
+    TweenLite.to(textClock, 0, { autoAlpha: 0 });
+}
+function free() {
+    OBSERV.innerHTML = "";
+}
+function serialVisible1() {
+    TweenLite.to(textI1, 0, { autoAlpha: 1 });
+}
+function serialVisible2() {
+    TweenLite.to(textI2, 0, { autoAlpha: 1 });
+}
+function serialVisible3() {
+    TweenLite.to(textI3, 0, { autoAlpha: 1 });
+}
+function serialVisible4() {
+    TweenLite.to(textI4, 0, { autoAlpha: 1 });
+}
+function clockVisible() {
+    TweenLite.to(textClock, 0, { autoAlpha: 1 });
+}
+
+function selectDisappear() {
+    TweenLite.to(textSelect, 0, { autoAlpha: 0 });
+    gsap.set(SELECT, {
+
+        fill: "#008000"
+    });
+selectTextDisappear();
+
+}
+selectDisappear();
+function selectVisibleTo1() {
+    TweenLite.to(textSelect, 0, { autoAlpha: 1 });
+    gsap.set(SELECT, {
+
+        fill: "#29e"
+    });
+    textSelect.textContent=1;
+    selectTextVisible();
+    
+
+}
+function selectVisibleTo0() {
+    TweenLite.to(textSelect, 0, { autoAlpha: 1 });
+    gsap.set(SELECT, {
+
+        fill: "#eeeb22"
+    });
+    textSelect.textContent=0;
+    selectTextVisible();
+}
+
+function allDisappear() {
+    serialDisappear1();
+    serialDisappear2();
+    serialDisappear3();
+    serialDisappear4();
+    serialDotDisappear();
+    selectDisappear();
+    clockDisappear();
+    clockDotDisappear();
+    outputDisappear();
+    gsap.set(I1, {
```

Comment:
In place of calling the set function umpteen times , can we just create an array of all these elements and use a forEach or for . of .. loop to set all of them to get much readable code?